kalite/main/content_rating_models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+from kalite.facility.models import FacilityUser
+
+from securesync.models import DeferredCountSyncedModel
+
+class ContentRating(DeferredCountSyncedModel):

```

Comment:
Need to add `minversion = "0.15.0"` in here (top-level within the class definition), ala https://github.com/MCGallaspy/ka-lite/blob/beautify-content-rating/kalite/main/models.py#L491